src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -174,152 +157,150 @@ void EnDns_Destroy(Actor* thisx, PlayState* play) {
 }
 
 void EnDns_ChangeAnim(EnDns* this, u8 index) {
-    s16 frameCount;
-
-    frameCount = Animation_GetLastFrame(sAnimationInfo[index].animation);
-    this->unk_2BA = index; // Not used anywhere else?
-    Animation_Change(&this->skelAnime, sAnimationInfo[index].animation, 1.0f, 0.0f, (f32)frameCount,
+    s16 frameCount = Animation_GetLastFrame(sAnimationInfo[index].animation);
+    this->lastAnimation = index; // Not used anywhere else?
```

Comment:
```suggestion
    s16 frameCount = Animation_GetLastFrame(sAnimationInfo[index].animation);
    
    this->lastAnimation = index; // Not used anywhere else?
```
as a style thing in the repo, we have an empty line after declarations